app/Models/FormAuth.php

```diff
@@ -71,7 +71,7 @@ public static function deleteCookie(): void {
 			@unlink(DATA_PATH . '/tokens/' . $token . '.txt');
 		}
 
-		if (rand(0, 10) === 1) {
+		if (random_int(0, 10) === 1) {
```

Comment:
Waste of energy: no need to have an expensive RNG here

Reply:
https://stackoverflow.com/questions/44228718/php-rand-vs-random-int
ok but i dislike rand()

Reply:
What is your issue with `rand()`? Your link indicates that it is the best one to use there ;-)
A Mersenne Twister RNG is very fine here